build/Jenkinsfile.branch

```diff
@@ -60,26 +60,6 @@ pipeline {
             stash includes: 'src/github.com/mattermost/mattermost-server/dist/**', name: 'eedist'
           }
         }
-        stage ('Build TE') {
```

Comment:
i think is fine to not build the TE. wdyt @jwilander

Reply:
Yeah we can skip TE for sure 👍